src/keycloak/chart/templates/keycloak-logging-recording-rules.yaml

```
@@ -71,3 +72,4 @@ data:
                 )
               )
 {{- end }}
+# @lulaEnd a5540198-a55e-4769-a5de-44ffe7636555
```

nit: align indent with other comment
```suggestion
  # @lulaEnd a5540198-a55e-4769-a5de-44ffe7636555
```